### ISSUE TITLE
NVDA does not provide any way to read a disabled option in a listbox

### ISSUE THREAD

**Issue by zarend** [nvaccess/nvda#14190]
### Steps to reproduce:
1. Navigate to https://output.jsbin.com/zijumor
2. (Demo opens and page says "Select a color. 'Blue' is deliberately disabled.")
3. Tab to the select element and press space
4. (select options "red", "Green" and "Blue appear")
5. Press Down arrow
6. ("Green" option is browser and screen reader focused. NVDA announced "Green 2 of 3")
7 Press down arrow again

### Actual behavior:
Nothing happens

### Expected behavior:
I would expect there to be some way to read the third option in the listbox. I tried looking for another way to do this, and there is no way to read the third option. The "Blue" option is only communicated visually and there is no communication to screen reader that the third option is Blue.

### NVDA logs, crash dumps and other attachments:
N/A

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2021.3.1

#### Windows version:
Windows 10 Enterprise 20H2

#### Name and version of other software in use when reproducing the issue:
Chrome Version 105.0.5195.127 (Official Build) (64-bit)

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

### Additional Information
I'm running into this problem when implementing listbox components for the Angular Material component library. https://github.com/angular/components.

### Source code of example:

```html
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width">
  <title>JS Bin</title>
</head>
<body>

  <form>
    <p>Select a color. 'Blue' is deliberately disabled.</p>

    <select>
      <option>Red</option>
      <option>Green</option>
      <option disabled>Blue</option>
    </select>
  </form>
</body>
</html>
```

**Comment by rperez030** [comment]
I think blue not receiving focus is a browser behavior related to the use of the HTML disabled attribute. That I would expect. the real issue here is that NVDA is not exposing the disabled option even through object navigation.

1. Move the browse cursor to the combo box and press space bar to activate. NVDA will switch to focus mode.
2. Press the space bar again or Alt +Down arrow to expand.
3. Use the command to move to next object (Desktop Insert +Numpad 6) to review the content of the combo box. NVDA will announce Green not selected 2 of 3, followed by no next even though blue is available in the accessibility tree. I would expect NVDA to announce something like blue disabled 3 of 3.

The same issue can be observed in the new Windows 11 notepad, for example.

1. In Notepad, activate the Edit menu by pressing Alt +E.
2. Try reading the list of options with object navigation.

NVDA skips all the disabled items in the list rather than exposing them with the disabled state.

**Comment by JAWS-test** [comment]
Related: https://github.com/FreedomScientific/VFO-standards-support/issues/670

**Comment by rperez030** [comment]
@Brian1Gaff we could spent time debating design considerations around disabled elements or whether they should or should not receive keyboard focus. All of that is very controversial. The fact is that disabled elements are often part of the UI, and although not actionable, they convey information. In fact, they are exposed by NVDA in browse mode as well as when they receive focus directly. In the example provided, we know there is a third option because NVDA announces position information. The third option is visible on the screen but not accessible to the NVDA user. I wouldn't expect keyboard focus to go there because that is controlled by the browser, not NVDA, but NVDA's object navigator should go there. As an NVDA user, I expect to be able to access information about disabled controls.

@zarend , although I think this issue should be fixed, I would suggest not to use the HTML disabled attribute. You can use aria-disabled instead, and CSS to convey the disabled state visually.

**Comment by Adriani90** [comment]
This was fixed in #15785. Closing as works for me.
